### PR TITLE
retail install sections cleanup

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/8364

### CODE REVIEW

File: modules/retail/pages/retail-install-setup.adoc [hunk 1]
```diff
@@ -72,16 +72,16 @@ Manually create these system groups during installation:
...
-Navigate to menu:Systems[System Groups] and click on btn:[Create System Group].
+Navigate to menu:Systems[System Groups] and click btn:[Create System Group].
 
 For more information about system groups, see xref:reference:systems/system-groups.adoc[].
 
 
 
 [[retail.sect.install.branch]]
-== Install the branch server and register it to {susemgr}
+== Install the Branch Server and Register It to {susemgr}
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-setup.adoc]:
```suggestion
== Install the Branch Server and Register it to {susemgr}
```

File: modules/retail/pages/retail-install-setup.adoc [hunk 2]
```diff
@@ -91,7 +91,7 @@ Do not install the branch server as a traditionally managed client.
 
 For instructions to install Salt-based proxy clients and register them to {susemgr}, see xref:installation:install-proxy-unified.adoc[].
 
-The activation key should have the following channels, pools and respective updates:
+The activation key should have the following channels, pools, and respective updates:
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-setup.adoc]:
```suggestion
The activation key should have these channels, pools, and respective updates:
```

File: modules/retail/pages/retail-install-setup.adoc [hunk 3]
```diff
@@ -107,23 +107,26 @@ For mass deployments, see xref:retail:retail-mass-config.adoc[].
...
+== Assign and Configure Branch Server Formulas
 
 Before you configure the branch server, ensure you have decided on networking topology, and know the Salt ID of the branch server.
 
 The branch server can be configured automatically using the [command]``retail_branch_init`` command, as shown in this section.
 If you prefer to manually configure the branch server, you can do so using formulas.
 For more information about formulas, see xref:retail:retail-formulas.adoc[].
 
-.Procedure: Configuring branch server formulas using helper script
+.Procedure: Configuring Branch Server Formulas Using Helper Script
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-setup.adoc]:
```suggestion
.Procedure: Configuring Branch Server Formulas With a Helper Script
```

File: modules/retail/pages/retail-install-setup.adoc [hunk 4]
```diff
@@ -134,12 +137,12 @@ salt <branch_server_salt_id> state.apply
 
 
 
-== Synchronize images to the branch servers
+== Synchronize Images to the Branch Servers
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-setup.adoc]:
```suggestion
== Synchronize Images to the Branch Server
```

File: modules/retail/pages/retail-install-setup.adoc [hunk 5]
```diff
@@ -134,12 +137,12 @@ salt <branch_server_salt_id> state.apply
...
 
 The OS image you use on the {susemgr} server must be synchronized for use on the branch server.
 You can do this with the Salt [command]``image-sync`` tool.
 
-.Procedure: Synchronize images with branch server
+.Procedure: Synchronize Images with Branch Server
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-setup.adoc]:
```suggestion
.Procedure: Synchronizing Images to the Branch Server
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 6]
```diff
@@ -38,15 +38,15 @@ Review the settings and click btn:[Install].
 yast susemanager_setup
 ----
 . Follow the prompts to set up your account.
-Ensure you take note of the passwords you set, you will need them later on. 
+Ensure you take note of the passwords you set, you will need them later on.
 
 
 
-.Procedure: Installing {susemgr} Branch Server from a DVD image
+.Procedure: Installing {susemgr} Branch Server from a DVD Image
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
.Procedure: Installing the Branch Server from a DVD Image
```

Just to try and make it more different from L12

**Comment by keichwa** [comment]
Requested changes are in. The github UI does not seem to see this...  Thus, please review again, @Loquacity --or must I click a very special button?

**Pull request merged**